ADB_world/Worldmap_year/OneYearMap.py

```diff
@@ -0,0 +1,163 @@
+
+#######################################
+# Plotting ############################
+#######################################
+
+# Process flight connection data for the entire year
+flight_data_df, daily_flights_df = def_flight_connection.process_flight_connections("Year")
+
+# Calculate the maximum value of 'daily_flights' column
+max_daily_flights = daily_flights_df['number_of_total_flights'].max()
+
+# number of total connections
+total_connections = len(flight_data_df)
+
+# Define maximum and minimum sizes for markers
+max_size = 20
+min_size = 2
+
+# scale the values between min and max sizes
+scaled_sizes = ((daily_flights_df['number_of_total_flights'] - daily_flights_df['number_of_total_flights'].min()) /
+                (daily_flights_df['number_of_total_flights'].max() - daily_flights_df['number_of_total_flights'].min()) *
+                (max_size - min_size)) + min_size
+
+# Plot the different airport markers
+fig = go.Figure(go.Scattergeo(
+    lat=daily_flights_df["lat_departure"],
+    lon=daily_flights_df["lon_departure"],
+    text=daily_flights_df["departure_airport_name"] + '<br>'
+    + "Number of average departing daily flights: "
+    + daily_flights_df["number_of_total_flights"].apply(lambda x: round(x, 2)).astype(str),  # Round to 2 decimal places
+    hoverinfo='text',  # Set hoverinfo to 'text'
+    mode='markers',
+    marker=dict(
+        size=scaled_sizes,  # Scale the sizes between min and max sizes
+        opacity=0.9,
+        line_width=0,
+        autocolorscale=False,
+        colorscale='Bluered',
+        cmin=0,
+        color=daily_flights_df['number_of_total_flights'],
+        cmax=max_daily_flights,  # Use the calculated maximum value
+        colorbar_title="Average Daily<br>Departing Flights<br>per year"
+    )
+))
+
+# Define maximum and minimum opacities for flight connections
+max_opacity = 1
+min_opacity = 0.0
+
+# Scale the opacities between min and max opacities
+scaled_opacities = ((flight_data_df['averageDailyFlights'] - flight_data_df['averageDailyFlights'].min()) /
+                    (flight_data_df['averageDailyFlights'].max() - flight_data_df['averageDailyFlights'].min()) *
+                    (max_opacity - min_opacity)) + min_opacity
+
+# Plot all the different connections
+flight_paths = []
+for i in range(len(flight_data_df)):
+    fig.add_trace(
+        go.Scattergeo(
+            lon=[flight_data_df['lon_departure'][i], flight_data_df['lon_destination'][i]],
+            lat=[flight_data_df['lat_departure'][i], flight_data_df['lat_destination'][i]],
+            mode='lines',
+            line=dict(width=1, color='white'),
+            opacity=scaled_opacities[i],  # Use scaled opacities
+            hoverinfo='skip',
+        )
+    )
+
+# Update layout of the figure
+fig.update_layout(
+    title=f'Air Traffic Worldmap<br>Number of total different connections: {total_connections}',
+    showlegend=False,
+)
+
+# Update layout of the geospatial plot
+fig.update_geos(
```

Comment:
Where can I set the actual dimensions of the figure? Or is this simply set by the dimensions of planet earth (ie. the map)?